docs/kuneiform/procedures.mdx

```diff
@@ -0,0 +1,336 @@
+---
+sidebar_position: 3
+sidebar_label: "Procedures"
+id: kuneiform-procedures
+title: Kuneiform Procedures
+description: An outline of Kuneiform procedures.
+slug: /kuneiform/procedures
+---
+
+Kuneiform supports a basic procedural language for building logic into schemas. Procedures can be used for building access
+control logic, performing arithmetic, and managing control flow.
+
+## Declaring Procedures
+
+Procedures are declared using the `procedure` keyword. They can be given a list of parameters and their corresponding types,
+return types, and access modifiers.
```

Comment:
Add a note that if a procedure returns data, the return must be declared in the signature?

Reply:
I feel like that is sort've implied...

I'll add a note about strong typing